scripts/xslt/cdrh_to_html/lib/html_formatting.xsl

```diff
@@ -348,7 +348,7 @@
   </div>
 </xsl:template>
 
-<xsl:template match="hi[@rend='italic'] | hi[@rend='italics']" priority="1">
+<xsl:template match="hi[@rend='italic'] | hi[@rend='italics']" priority="2">
```

Comment:
would you mind adding a comment above this explaining that it's overriding the generic `@rend` match on 307?  I have a feeling that in the future I won't remember why the priority has been bumped.  We might also want to check if any other templates (lewis and clark, oscys) had overridden the rends using a priority, because I'm worried this might override their overrides, haha.